packages/x-date-pickers/src/PickersDay/PickersDay.tsx

```diff
@@ -19,11 +19,28 @@ import {
   pickersDayClasses,
 } from './pickersDayClasses';
 
+export interface ExportedPickersDayProps {
+  /**
+   * If `true`, today's date is rendering without highlighting with circle.
```

Comment:
Maybe it's not worth going into the styling specifics in JSDoc? 🤔 
```suggestion
   * If `true`, today's date is rendered without highlight.
```